flicker/flicker.py

```diff
@@ -985,7 +1000,11 @@ def value_counts(self, names, normalize=False, sort=True, ascending=False,
         if isinstance(names, six.string_types):
             names = [names]
         for name in names:
```

Comment:
Could you use the `_validate_column_name` method here too?

Reply:
Thanks!